webpack.config.js

```diff
@@ -0,0 +1,40 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+module.exports = {
+    // webpack will take the files from ./src/index
+    entry: './src/index',
+
+    // and output it into /dist as bundle.js
+    output: {
+        path: path.join(__dirname, '/dist'),
+        filename: 'bundle.js',
```

Comment:
Probably better to use `filename : 'bundle.[hash].js'` so that user won't have to hard reload for changes to take effect

Reply:
Changed in 6e66da9d377e7dbd1846ab86eb016da26ef1b156